frontend/public/components/_masthead.scss

```diff
@@ -1,3 +1,56 @@
+.co-app-launcher {
+  ul {
+    list-style: none;
+    margin-bottom: 0;
+    padding-left: 0;
+  }
+
+  .pf-c-app-launcher__menu-item-external-icon {
+    --pf-c-app-launcher__menu-item-external-icon--Color: var(--pf-global--icon--Color--light);
+
+    opacity: 1;
+  }
+}
+
+.co-cluster-menu {
+  max-width: 80vw;
+
+  @media (min-width: $screen-sm) {
+    max-width: 90vw;
+  }
+}
+
+.co-cluster-selector {
+  display: inline-grid !important;
+  grid-template-columns: 1fr auto;
+  max-width: 140px !important;
+
+  @media (min-width: 360px) {
+    max-width: 180px !important;
+  }
+
+  @media (min-width: $screen-xs) {
+    max-width: 300px !important;
+  }
+
+  @media (min-width: $screen-lg) {
+    max-width: 400px !important;
+  }
+
+  // PatternFly does not fully support <MenuToggle isFullHeight> styling in <Masthead>
+  &::before {
+    border: 1px solid var(--pf-global--palette--black-800) !important;
+    border-bottom: 0 !important;
+    border-top: 0 !important;
+    bottom: 0;
+    content: "";
+    left: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
```

Comment:
```suggestion
  &::before {
    bottom: 0;
    content: "";
    left: 0;
    position: absolute;
    right: 0;
    top: 0;
  }
  
  &.pf-c-menu-toggle {
    --pf-c-menu-toggle--after--BorderBottomColor: transparent;
  }
```

Reply:
`--pf-global--palette--black-800` doesn't contrast on the dark theme header. The default menu toggle borders seem to work better.

Reply:
Updated to add a better border color for dark mode.